Auth info (#99)
vippsas/vipps-report-api: api-guide/overview.md

## Sales units that do not have API access

> It is not possible for a merchants to use the Report API directly with Vippsnummer sales units,
> since there is no API access.
> 

Reviewer: ```suggestion
It is not possible for merchants to use the Report API directly with Vippsnummer sales units,
since the sales unit doesn't have API access.

```

Reviewer: Also, Vippsnummer vs vm number here?
